internal/page/testing_start.go

```diff
@@ -247,6 +248,21 @@ func TestingStart(store sesh.Store, donorStore DonorStore, randomString func(int
 
 			if opts.hasDonorDetails {
 				lpa.Donor = makeDonor()
+				if opts.cannotSign {
+					lpa.Donor.ThinksCanSign = actor.No
+					lpa.Donor.CanSign = form.No
+
+					lpa.AuthorisedSignatory = actor.AuthorisedSignatory{
+						FirstNames: "Alan",
+						LastName:   "Signatory",
+					}
+
+					lpa.IndependentWitness = actor.IndependentWitness{
+						FirstNames: "Iris",
+						LastName:   "Witness",
+					}
```

Comment:
There were some names suggested in the [UR doc](https://docs.google.com/document/d/1nT0pj_BZbVeDcG2TI13pX24wcJ6QZU7BfrTEa_NByOs/edit) recently for these two roles but I couldn't add them as they didn't exist in code - worth updating now to save future changes.

Reply:
Oh looks like that doc may be locked down. The names were:

Independent witness: Indie Irwin
Authorised signatory: Allie Adams